amy/dashboard/views.py

```diff
@@ -209,8 +216,6 @@ def autoupdate_profile(request):
 
 @login_required
 def training_progress(request):
```

Comment:
Given how it has been simplified, it should be turned into a generic `TemplateView`. Can be done later as long as there's a ticket for it.

Reply:
Good point, made #2503